cmd/chain-receipts/main.go

```diff
@@ -100,12 +100,10 @@ func listener(provider *ethrpc.Provider, monitorOptions ethmonitor.Options, rece
 	// Find specific meta transaction -- note: this is not the "transaction hash",
 	// this is a sub-transaction where the id is emitted as an event.
 	FilterMetaTransactionID := func(metaTxnID ethkit.Hash) ethreceipts.FilterQuery {
-		return ethreceipts.FilterLogs(func(logs []*types.Log) bool {
```

Comment:
can you write a little benchmark separately to see what the performance effects are of doing this..? I'm curious if this will save us memory in practice..?

Reply:
After writing a benchmark neither pointer removal or sonic help with performance.